src/App.scss

```diff
@@ -46,6 +46,26 @@
   .header_nav__inner {
     background: rgba($gray-light, 0.5);
   }
+
+  .footer {
+    position: relative;
+    height: 0;
```

Comment:
Why footer has zero height?

Reply:
Because it incorrectly calculates with filter blur and causes rendering issues